src/components/commons/sider/sider-body.tsx

```diff
@@ -58,39 +58,50 @@ const SiderBody = () => {
 	}, [realms, dispatch]);
 
 	useEffect(() => {
-		const match = matchPath(location.pathname, [
-			'/realm/:realm/us/:userStorage',
-			'/realm/:realm/',
-		]);
+		const match = matchPath<{ realm: 'string'; userStorage: 'string' }>(
```

Comment:
Do you think it's possible to extract this code from the sider-body component and put it in the root component (app.tsx) ? You can pass some params and function from the app component by the props (i don't think that use redux could be a good idea for the moment)

Reply:
the goal is to have a pure ui component